autoware_perception_msgs/msg/TrafficLightElement.msg

```diff
@@ -0,0 +1,29 @@
+# constants for color
+uint8 RED = 1
+uint8 AMBER = 2
+uint8 GREEN = 3
+uint8 WHITE = 4
```

Comment:
What is white color for?

Reply:
I also don't know the use case. It's for compatibility with `Trafficlight.idl`.

Reply:
I asked a colleague and found out that it is based on [Vienna Convention](https://en.wikipedia.org/wiki/Vienna_Convention_on_Road_Signs_and_Signals).
This is also mentioned in the message addition PR.
https://gitlab.com/autowarefoundation/autoware.auto/autoware_auto_msgs/-/merge_requests/16

Reply:
@isamu-takagi thank you so much :heart: , maybe you can include vienna convention link in the readme too.